packages/react/src/modules/guide/components/GuideToolbar/GuideToolbar.tsx

```diff
@@ -15,8 +16,19 @@ export const GuideToolbar = () => {
   const [isCollapsed, setIsCollapsed] = useState(false);
 
   const { client } = useGuideContext();
+
   const debugState = useStore(client.store, (state) => state.debug);
 
+  const isGuideMissing = useStore(client.store, (state) => {
+    return (
+      !!state.debug.forcedGuideKey &&
+      !(
+        state.guides[state.debug.forcedGuideKey] ||
+        state.previewGuides[state.debug.forcedGuideKey]
+      )
```

Comment:
I think we could check whether 1) client.stage.status is closed and 2) this.stage.resolved matches the forcedGuideKey? If forcedGuideKey is set, and the group stage is closed, but we were not able to resolve the guide - then I think that must mean there was no useGuide() looking for that guide? We'd need to make the `stage` property public.